core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -109,7 +109,7 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     zkUtils = ZkUtils(zkUrl,
                       sessionTimeout = zkSessionTimeOutMs,
                       connectionTimeout = zkConnectionTimeoutMs,
-                      JaasUtils.isZkSecurityEnabled())
+                      kafkaConfig.zkEnableSecureAcls)
```

Comment:
We need to check that this doesn't cause issues when `SimpleAclAuthorizer` is used by `AclCommand`. We may have to set this config in `AclCommand`.